src/components/card/card.css

```diff
@@ -0,0 +1,144 @@
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 32px 56px;
+  gap: 24px;
+  background-color: #ffffff;
+}
+
+.small {
+  background: #ffffff;
+  border-radius: 20px;
+  font-family: 'Nunito Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 40px;
+  gap: 40px;
+}
+
+.profile-image {
```

Comment:
remove unnecesary styles that are not related to Card component move it to utils/utils.css 

please check figma desing card is not similar at all